website/docs/d/iam_trusted_profiles.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profile"
+description: |-
+  Get information about iam_trusted_profile
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profile
+
+Provides a read-only data source for iam_trusted_profile. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax. For more information about trusted profile, see https://cloud.ibm.com/apidocs/iam-identity-token-api#create-profile
```

Comment:
Provides a read-only data source for iam_trusted_profile. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax. For more information about trusted profile, see https://cloud.ibm.com/apidocs/iam-identity-token-api#create-profile

as

Retrieve information about IAM trusted profile as a read-only data source. For more information, about trusted profile, see [created a trusted profile](https://cloud.ibm.com/apidocs/iam-identity-token-api#create-profile).